.bids_conversion/convert.sh

```diff
@@ -0,0 +1,192 @@
+    git mv -fk ${subdir}/T1w/T2w_acpc_dc.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dc_T2w.nii.gz || true
+    git mv -fk ${subdir}/T1w/T2w_acpc_dc_restore.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dcrestore_T2w.nii.gz || true
+
+    # rename func files
+    #
+    # 3T rest1 LR
+    #
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_AbsoluteRMS_mean.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementAbsoluteRMSmean.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_AbsoluteRMS.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementAbsoluteRMS.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_Regressors_dt.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRegressorsdt.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_Regressors.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRegressors.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_RelativeRMS_mean.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRelativeRMSmean.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_RelativeRMS.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRelativeRMS.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsNoise.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-ReclassifyAsNoise.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsSignal.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir_LR_run-1_desc-ReclassifyAsSignal.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_hp2000_clean.dtseries.nii ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-Atlashp2000clean.dtseries.nii || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_MSMAll_hp2000_clean.dtseries.nii ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-AtlasMSMALLhp2000clean.dtseries.nii || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_stats.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-Atlasstats.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_CSF.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-CSF.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_hp2000_clean.nii.gz ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-hp2000clean.nii.gz || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Physio_log.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_physio.txt || true
```

Comment:
checked -- this one is a .tsv (no header). BIDS likes .tsv. 
Interestingly, BIDS seems to demands them to be `.gz`'ed!  https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/06-physiological-and-other-continuous-recordings.html

But I think this should be sensible to ask to rectify: https://github.com/bids-standard/bids-specification/issues/472

Reply:
anyone (@loj @adswa) would be interested to become a BIDS contributor and propose a PR which would make `.gz` optional for these files? There seems to be no negative votes so far in the BIDS issue ;)